src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -58,11 +64,28 @@ const props = defineProps({
     type: String,
     default: 'Clear filters',
   },
+  areFiltersHidden: {
+    type: Boolean,
+    default: () => undefined,
+  },
+  hiddenFiltersNames: {
+    type: Array,
+    default: () => [],
+  },
 });
 
 const emit = defineEmits(['update:modelValue', 'change']);
+const areFiltersHiddenThreshold = useMediaQuery('(max-width: 768px)');
```

Comment:
I think the toggling of visibility should be handled by the parent component when used.